src/contrib/cluster/Akka.Cluster.Tools.Tests.MultiNode/ClusterClient/ClusterClientDiscoverySpec.cs

```diff
@@ -107,8 +107,6 @@ protected ClusterClientDiscoverySpec(ClusterClientDiscoverySpecConfig config)
             _config = config;
         }
 
-        protected override int InitialParticipantsValueFactory => 3;
```

Comment:
This value should be set to `4`, which the test will do by default. With a value of `3` it's possible that the `RoleName("third")` will start up late and hit a bunch of barriers late, causing the coordinator node to crash.